hasura.planx.uk/migrations/1683874331828_create_table_public_email_applications/up.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE "public"."email_applications" (
+  "id" serial NOT NULL, 
+  "session_id" text NOT NULL, 
```

Comment:
If we make this a foreign key to `lowcal_sessions` this should have the added benefit of cascading deletes as a fallback? This is how `payment_request` is set up.

```sql
FOREIGN KEY ("session_id") REFERENCES "public"."lowcal_sessions"("id") ON UPDATE restrict ON DELETE cascade
```